### ISSUE TITLE
Sabre picks incorrect compiler version

### ISSUE THREAD

**Issue by b-mueller** [b-mueller/sabre#47]
When pragma is set to a fixed version, Sabre attempts to compile with a later release which fails. E.g.:

suicide.sol:
```
pragma solidity 0.4.24;

contract Suicide {

  function kill(address addr) public {
    if (addr == address(0x0)) {
      selfdestruct(addr);
    }
  }

}
```

Output:

```
$ sabre solidity_examples/suicide.sol 
✔ Compiled with solc v0.4.25 successfully
inputfile:1:1: SyntaxError: Source file requires different compiler version (current compiler is 0.4.25+commit.59dbf8f1.Emscripten.clang - note that nightly builds are considered to be strictly less than the released version
pragma solidity 0.4.24;
^---------------------^
```

**Comment by Skyge** [comment]
I will have a look.

**Comment by Skyge** [comment]
OK, I have fixed it, please wait for a moment.

**Comment by Skyge** [comment]
I have tested 
 - 0.4.23
 - ^0.4.23
 - >=0.4.23
 - >=0.4.23<=0.5.0
 - >=0.4.23<0.5.0
 - <=0.5.0
 - <0.5.0
 - <=0.6.0
 - <0.6.0
 - without specific version

**Comment by b-mueller** [comment]
Fixed by #48.
